kolibri/core/tasks/job.py

```diff
@@ -277,26 +277,28 @@ def add(x, y):
         Look at each method's docstring for more info.
     """
 
+    from kolibri.core.tasks.queue import DEFAULT_QUEUE
+
     def __init__(
         self,
         func,
-        validator,
```

Comment:
This seems to have been reverted from the previous PR?

Reply:
No sir, actually I first removed the default values of the parameters but then I added them back so that instances of `RegisteredJob` can be created by passing only a part of the arguments. Might help in testing.

Reply:
Yes, I agree that having the defaults is helpful, I was just curious as you had previously had defaults, removed them, and now they were coming back in. I think this change is good, just wanted to make sure it was intentional.